web-backend/metrics/src/rust/cargoaudit.rs

```diff
@@ -23,6 +23,9 @@ pub async fn audit(repo_path: &Path) -> Result<Report> {
     let advisory_db = rustsec::Database::load_from_repo(&advisory_db_repo)
         .with_context(|| "couldn't open RUSTSEC repo")?;
 
+    // TODO: rm -rf advisory_db_path
+    // rationale: if once the command fails or get interrupted, the path gets damaged, and fetch fails afterwards everytime
+
     // make sure a Carg.lock file is there
```

Comment:
The issue is discussed here: https://github.com/RustSec/rustsec/issues/32. The issue may have been fixed in the latest version. However, that may break a lot of things. -_-